andela/ah-backend-athena
# #162163287 user should be able to view the reading statistics of an article

**What does this PR do?**
- Allows an author to view the reading statistics of his/her article
**Description of Task to be completed?**
- An author can easily view the reading statistics of his article when he hit on this endpoint 
`api/articles/: slug/time`. For backend, the time a user is expected to spend on an endpont is passed in the endpoint which is compared with the article read time to evaluate a read of a click without read.
- An *article read* is when the time spent on an endpoint is equal or greater than the article read time.
- Users who spend less than  the time required to read an article on a page are not counted as viewers.
**How can this be manually tested?**
- pass the slug of the article to read in the endpoint and also add the time a user is expected to the endpoint into the endpoint `POST http://127.0.0.1:8000/api/articles/todays-demo-38dc28b3c16b/10`
`output`
<img width="540" alt="screen shot 2018-12-19 at 17 23 35" src="https://user-images.githubusercontent.com/32167860/50225927-e7d3f300-03b2-11e9-8e3f-3bd60c7f7075.png">

- perform a get on `api/articles/: slug/` to view these statistics from the article
<img width="663" alt="screen shot 2018-12-19 at 17 22 56" src="https://user-images.githubusercontent.com/32167860/50225944-f7533c00-03b2-11e9-8814-9cebdeef37e1.png">


**Relevant story**
- #162163287
